observability/micrometer/src/main/resources/schema/wildfly-micrometer_1_0.xsd

```diff
@@ -1,5 +1,5 @@
 <!--
-  ~ Copyright 2022 Red Hat, Inc.
+  ~ Copyright 2022-2023 Red Hat, Inc.
```

Comment:
No need to change, but don't bother updating these dates. We've been told by those who know that this isn't necessary.